### ISSUE TITLE
QUOTE not defined

### ISSUE THREAD

**Issue by sumitkoundilya** [SheetJS/sheetjs#2820]
![Capture](https://user-images.githubusercontent.com/32324157/232846821-bc6e8659-3c47-4843-91e4-e224a16e5d2e.PNG)
Angular CLI: 14.2.11
Node: 14.21.3
Package Manager: npm 6.14.18
rxjs                            7.5.7
typescript                      4.7.4

**Comment by sumitkoundilya** [comment]
![Capture](https://user-images.githubusercontent.com/32324157/232847523-79be56fb-8f20-4b35-a97b-01e324d447fc.PNG)
Is this because of semicolon?
var APOS = "'"; QUOTE = '"' 
to 
var APOS = "'", QUOTE = '"'

**Comment by tusharrana786** [comment]
@sumitkoundilya yeah it is due to semicolon and semicolon should be replaced by ,

**Comment by johnLamberts** [comment]
It's not working though

**Comment by philip-firstorder** [comment]
Getting also this error, is it solved?

**Comment by johnLamberts** [comment]
Yes, go to the documentation. I literally `npm install` the cdn links from there. 

I mean, in my case. I already tried their solution but I still go back to the documentation then I found my solution anw.

**Comment by WiharlleyWill** [comment]
Use XLSX lib.

`import * as XLSX from 'xlsx';
...

exportToExcel() {
const fileName = 'test.xlsx';

	const ws: XLSX.WorkSheet = XLSX.utils.json_to_sheet(this.arr);
	const wb: XLSX.WorkBook = XLSX.utils.book_new();
	XLSX.utils.book_append_sheet(wb, ws, 'test');

	XLSX.writeFile(wb, fileName);
}
`

**Comment by SweydManaf** [comment]
has vulnerability

**Comment by WiharlleyWill** [comment]
> has vulnerability

report where there is vulnerability. It is not enough to just affirm.

**Comment by sumitkoundilya** [comment]
> It's not working though

Did you try changing in node modules builds?
I replaced that semicolon to coma, and it worked, but this is only useful when you have a project for local purpose. 🙂

**Comment by AlexD1979** [comment]
Is this issue now solved in a new version, does a PR exists?

**Comment by John-Fixit** [comment]
> Is this issue now solved in a new version, does a PR exists?
Don't think so, because am having the same error currently

**Comment by gustavosilva7** [comment]
same here

**Comment by pedroluanx28** [comment]
same here

**Comment by sagarpatelpro** [comment]
I have a work around.
Just declare "QUOTE" as global variable. 

In my case, I am writing a react application, so I just added it to the index.html file.

In index.html file, before including any other source code script.
`<script type="text/javascript">
    var QUOTE;
</script>`
